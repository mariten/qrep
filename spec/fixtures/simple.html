
<!DOCTYPE html>
<html>
<head>
  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
  <title>Simple</title>
  <script type='text/javascript' src='/js/lib/dummy.js'></script>
  <link rel="stylesheet" type="text/css" href="/css/result-light.css">
  <style type='text/css'>
  </style>
  <script type='text/javascript'>//<![CDATA[
    window.onload=function(){
    }//]]>
  </script>
</head>
<body>
  <header>
    <span id="logo">logo</span>
</header>
<div id="main">
    <h1>h1</h1>
</div>
</body>
</html>
