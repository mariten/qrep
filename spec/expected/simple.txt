<span id="logo">logo</span>
